loading module bar from mod file Test/verify1_bar.mod
	adding to modules map: Test/verify1_bar.c -> bar
loading module foo from mod file Test/verify1_foo.mod
	adding to modules map: Test/verify1_foo.c -> foo
	adding to modules map: Test/verify1_foo.h -> foo
setting default module to default
warning: the -fo-merge-E_variable-and-var-values flag is incompatible with doing module analyses, so turning it off.
skipping function in default module: _OSSwapInt16
skipping function in default module: _OSSwapInt32
skipping function in default module: _OSSwapInt64
localize: alnum-name:struct_Foo_a7198b88, size:sta, c-type-name:struct Foo
--- Test/verify1_foo.c
+++ Test/verify1_foo.c
@@ -6,1 +6,1 @@
-  return (struct Foo*) malloc(sizeof(struct Foo));
+  return malloc_struct_Foo_a7198b88_foo();
